classes/options-general.php

```diff
@@ -821,20 +821,20 @@ class="radio-thumbnail<?php if ( 'TERRAIN' == $map_type ) {
 
 													<tr valign="top">
 														<td><label
-																for="map_stylesheet"><?php _e( 'Theme', 'simplemap' ); ?></label>
+																for="map_stylesheet"><?php _e( 'Theme', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 129 characters

<!-- Comment by [SideCI](https://sideci.com) -->